modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/APMAgentSettings.java

```diff
@@ -159,7 +167,7 @@ public void setAgentSetting(String key, String value) {
 
     public static final Setting<Boolean> APM_ENABLED_SETTING = Setting.boolSetting(
         APM_SETTING_PREFIX + "enabled",
-        false,
+        true,
```

Comment:
we should switch this back to false before we merge and override this in serverless

Reply:
Thanks for adding a commit to do this.